tests/e2e/key/suite.go

```diff
@@ -117,4 +121,61 @@ var _ = ginkgo.Describe("[Key]", func() {
 		_, err = commands.CreateKey(keyName)
 		gomega.Expect(err).Should(gomega.HaveOccurred())
 	})
+
+	ginkgo.It("can export a key to stdout", func() {
+		// Create key
+		output, err := commands.CreateKeyFromPath(keyName, testKey)
+		if err != nil {
+			fmt.Println(output)
+			utils.PrintStdErr(err)
+		}
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		// Export the key
+		exportedKey, err := commands.ExportKey(keyName)
+		if err != nil {
+			fmt.Println(exportedKey)
+			utils.PrintStdErr(err)
+		}
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		// Trim the trailing newline from the export
+		exportedKey = strings.TrimSuffix(exportedKey, "\n")
+
+		// Check two keys are equal
+		originalKeyBytes, err := os.ReadFile(testKey)
+		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(exportedKey).Should(gomega.Equal(string(originalKeyBytes)))
+	})
+
+	ginkgo.It("can export a key to file", func() {
+		// Check output key does not already exist
+		_, err := os.Stat(outputKey)
+		gomega.Expect(err).Should(gomega.HaveOccurred())
+
+		// Create key
+		output, err := commands.CreateKeyFromPath(keyName, testKey)
+		if err != nil {
+			fmt.Println(output)
+			utils.PrintStdErr(err)
+		}
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		// Export the key
+		output, err = commands.ExportKeyToFile(keyName, outputKey)
+		if err != nil {
+			fmt.Println(output)
+			utils.PrintStdErr(err)
+		}
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		// Check two keys are equal
+		equal, err := utils.CheckKeyEquality(testKey, outputKey)
+		if err != nil {
+			fmt.Println(output)
+			utils.PrintStdErr(err)
+		}
+		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(equal).Should(gomega.BeTrue())
```

Comment:
Shouldn't we remove the test output file afterwards?

Reply:
it gets cleaned up by the `AfterEach`